docs/_articles/en/codebuilder/cb-start.md

```diff
@@ -0,0 +1,225 @@
+1. From the App Launcher, find and open **Code Builder**.
+
+**Note:** If you don't see Code Builder as an option, or you see an empty screen when you launch it, contact your admin to make sure you have the correct license and permission set assigned to you.
+
+2. Click **Get Started**.
+3. Click **New Project** to create a new Salesforce DX project.
+4. Give your project a name and choose **Standard** for project type.
+
+Next, connect an org to your Code Builder environment. You can choose to connect to a Salesforce org or a sandbox org.
+
+1. Log in to your development environment, click **Allow** to grant access, then click **Next**.
+2. Choose an alias (nickname) for your org. If you work on many orgs, make sure the nickname helps you identify the org quickly in the future. 
+Your new Code Builder environment is now available on your dashboard.
+3. Click **Launch** to launch Code Builder in a new tab. 
+4. Now sit back and relax for a few minutes while Code Builder creates and configures your workspace. Code Builder might take a fews minutes to start up when it initially sets up.
+
+**Tip**: Once you have created your Code Builder environment, launch it from the dashboard at any time, or bookmark it for fast access.
+
+## Let's Take A Quick Tour
+You’re now in your developer environment in Code Builder. Code Builder has VS Code’s rich IDE plus it gives you easy access to cool Salesforce development-specific tools through the Salesforce extension pack. Let’s take a quick tour and get to know the lay of the land. Your screen looks something like this:
+
+![code_builder_ui](../../../images/code_builder_ui.jpg)
+The Code Builder user interface is divided into five main areas.
+
+**Activity Bar**: Located on the left-hand side, it contains iconic buttons to switch between different views. In our example, the Explorer view is active.
+
+**Side Bar**: Contains different views to assist you while working on your project. For example, you can see the Explorer view in our example. It lists the files in DX project structure in your project.
+
+**Editors**: The main areas to edit your files. You can open as many editors as you like side by side, vertically and horizontally.
+
+**Panel**: Shows output or debug information, errors and warnings, or an integrated terminal. You can choose the output to display.
+
+**Status Bar**: Shows information about the opened project and the files you’re editing. The Status Bar displays the org that you're connected to. 
+
+## Get To Know Your Org – Use the Org Browser
+Take a closer look at the activity bar and notice a cloud icon. This icon represents the Org Browser. It’s a part of the Salesforce Extension pack. It helps you browse and retrieve metadata from your org without having to use a manifest file. The Org Browser is available only in non source-tracked orgs such as sandboxes or Developer Edition orgs. User Org Browser to retrieve the metadata source into your local project. 
```

Comment:
```suggestion
Take a closer look at the activity bar and notice a cloud icon. This icon represents the Org Browser. It’s a part of the Salesforce Extension pack. It helps you browse and retrieve metadata from your org without having to use a manifest file. The Org Browser is available only in non source-tracked sandboxes or Developer Edition orgs. User Org Browser to retrieve the metadata source into your local project. 
```

Reply:
Suggesting a small tweak because you can enable source tracking in sandboxes and it's required for DevOps Center. But I feel like this wording could be misconstrued that all sandboxes don't have source tracking.